pkg/client/app.go

```diff
@@ -96,6 +107,15 @@ func ToAppUpdate(ctx context.Context, c Client, name string, opts *AppUpdateOpti
 	}
 
 	if opts.Image != "" {
+		img, tag, err := FindImage(ctx, c, opts.Image)
+		if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Here you are checking for `apierros.IsNotFound` and above you are checking `images.ErrImageNotFound{}`. Is this intentional?

Reply:
Absolutely not intentional. Thanks for the hint!